Makefile

```diff
@@ -9,7 +9,8 @@ CPP = gcc -E
 xcOK = 0
 
 xcCUDA = KEPLER
-xcCUDA = FERMI
```

Comment:
I don't have Fermi generation GPU, and the latest CUDA does not support Fermi GPUs